cmd/agent-js-worker/main.go

```diff
@@ -801,6 +801,110 @@ func decodeAndGunzip(zcap string) ([]byte, error) {
 	return uncompressed, nil
 }
 
+func capabilityInvocationAction(req *http.Request) (string, error) { //nolint:funlen,gocognit,gocyclo
+	s := strings.Split(req.URL.Path, "/")
+
+	const minPathLen = 5 // /v1/keystores/{key_store_id}/keys
+
+	if len(s) < minPathLen {
```

Comment:
path length is fixed?

Reply:
it has minimal number of components within it that is 5: `/v1/keystores/{key_store_id}/keys`, `/v1/keystores/{key_store_id}/easyopen`
but the full path might be longer, e.g. `/v1/keystores/{key_store_id}/keys/{key_id}` or `/v1/keystores/{key_store_id}/keys/{key_id}/sign`